packages/serialized-state-manager/README.md

```diff
@@ -0,0 +1,42 @@
+# @feature-hub/serialized-state-manager
+
+[![Package Version][package-badge]][package-npm]
+[![Website][website-badge]][website] [![API][api-badge]][api]
+
+A Feature Service to manage handing over state to the client when server-side
```

Comment:
Alternative proposal for the description:
`A Feature Service for managing the state transfer to the client when rendering on the server.`

Reply:
I like it. 👍